data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids.py

```diff
@@ -0,0 +1,104 @@
+# Project imports
+from constants.cdr_cleaner import clean_cdr as clean_consts
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from notebooks import bq
```

Comment:
`from notebooks import bq` this import is not correct, you probably need `import bq_utils` and please checkout `bq_utils.query`  for executing queries

Reply:
@ChaoPang this import uses the BigQuery client library. The `bq_utils` is the API, which I think we are eventually moving away from. Another reason I didn't want to use `bq_utils.query`, is because that executes the query and does not return the query results, since I just wanted to get the query results back, I opted for `bq` in notebooks.